### PR TITLE
Update geolocation documentation

### PR DESCRIPTION
**Pull request by edmundoa**
Update geolocation documentation to:

- Include Geo IP data adapter.
- Remove references to Map plugin.
- Update setup guide, as using a Lookup Table is now considered to be the way to go to get geolocation from your messages (we still provide the old Message processor, though).

### CODE REVIEW

**Review by jalogisch — APPROVED** [review]
WOW @edmundoa you are the king.

After reading this: https://blog.reconinfosec.com/geolocation-in-graylog/  I always wanted to point people in this direction, but did not find the time for that.

File: pages/geolocation.rst [hunk 1]
```diff
@@ -5,102 +5,78 @@ Geolocation
...
+.. note:: Before you get started, we recommend taking a look at some Lookup
+   Table concepts in :ref:`the documentation <lookuptables>`.
 
-.. note:: In case you are not sending structured logs to Graylog, you can use extractors to store the IP addresses in your messages into their own fields. Check out the :ref:`extractors` documentation for more information.
 
-.. important:: The GeoIP Resolver processor will **not process** any internal message fields, i. e. any field starting with ``gl2_`` such as ``gl2_remote_ip``.
+Download the database
+---------------------
 
+In first place, you need to download a geolocation database. The Lookup Table
```
Comment (dennisoelkers) [inline pages/geolocation.rst]: `In the first place`

File: pages/geolocation.rst [hunk 2]
```diff
@@ -5,102 +5,78 @@ Geolocation
...
+Geo IP Data Adapter supports both **MaxMind City and Country databases** in
+the **MaxMind DB format**, as the
+`GeoIP2 Databases <https://www.maxmind.com/en/geoip2-databases>`_ or
+`GeoLite2 Databases <https://dev.maxmind.com/geoip/geoip2/geolite2/>`_ that MaxMind provides.
 
-Verify the geolocation configuration (Optional)
------------------------------------------------
+The next step is to store the geolocation database in all servers running
```
Comment (dennisoelkers) [inline pages/geolocation.rst]: `on all servers`

**Review by dennisoelkers — CHANGES_REQUESTED** [review]
LGTM, just two minor grammar remarks.

**Comment by edmundoa** [comment]
Changed! Thank you for the review @jalogisch and @dennisoelkers! 🌮

**Review by dennisoelkers — APPROVED** [review]

**Pull request merged**